engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -102,7 +100,7 @@ trait Backend {
    * Essentially turns a Call object + CallInputs into a BackendCall
    */
   def bindCall(jobDescriptor: BackendCallJobDescriptor,
```

Comment:
The comments are not right, and in addition to its jarring name I don't think we need this anymore.  All this does in the three implementations is copy the supplied abort function into the descriptor.